### PR TITLE
Replacing pyviz with holoviz in the README file

### PR DESCRIPTION
**Pull request by kebowen730**
@philippjfr  @jlstevens

### CODE REVIEW

**Comment by jlstevens** [comment]
Personally, I think this is fine despite the inconsistency with the name in code. I'm happy for Philipp to override me if he feels `pyviz_comms` still has value outside our own ecoysystem.

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # pyviz_comms
 
-Offers a simple bidirectional communication architecture for PyViz tools
+Offers a simple bidirectional communication architecture for HoloViz tools
 including support for Jupyter comms in both the classic notebook and
 Jupyterlab.
```
Comment (jbednar) [inline README.md:5]:
```suggestion
Offers a simple bidirectional communication architecture between Python and JavaScript, 
with support for Jupyter comms in both the classic notebook and Jupyterlab.
Available for use by any [PyViz](https://pyviz.org) tool, but currently primarily used by
[HoloViz](https://holoviz.org) tools.
```

Comment (kebowen730) [reply README.md:5]: Looks like a good idea to me.

**Comment by jbednar** [comment]
I'm happy for this to be merged once the tests are passing.

**Comment by jbednar** [comment]
Tests are failing the same way on master; will merge this now but need to investigate that separately.

**Comment by philippjfr** [comment]
Master is passing just fine, this PR just wasn't rebased since it was fixed.

**Pull request merged**